server/controllers/api.py

```diff
@@ -672,6 +672,18 @@ def can(action, resource, requester):
             return True
         return resource == requester
 
+class CourseEnrollment(Resource):
+    """ Information about all students in a course.
+    Authenticated. Permissions: >= User or admins
+    Used by: Export scripts.
+    """
+    model = models.Enrollment
+    schema = EnrollmentSchema()
+
+    @marshal_with(schema.get_fields)
+    def get(self, cid, user):
+        return {'courses': self.model.get_all_students(cid)}
```

Comment:
This doesn't provide any kind of authentication/permissions right now. That means that any user can get the contact information for students for any course. 

```
if not self.model.can(course, user, 'staff'): 
    restful.abort(403)
```